data/scripts.js

```diff
@@ -3288,7 +3287,7 @@ exports.BattleScripts = {
 			item = 'Chople Berry';
 		} else if (ability === 'Iron Barbs' || ability === 'Rough Skin') {
 			item = 'Rocky Helmet';
-		} else if (counter.Physical + counter.Special >= 4 && ability === 'Regenerator' && template.baseStats[counter.Special >= 2 ? 'atk' : 'spa'] > 99 && template.baseStats.spe <= 80) {
+		} else if (counter.Physical + counter.Special >= 4 && ability === 'Regenerator' && template.baseStats[counter.Special >= 2 ? 'atk' : 'spa'] > 99 && template.baseStats.spe <= 80 && !hasMove['wideguard']) {
```

Comment:
Is AV/Wide Guard even a reported issue? This logic block dictates that AV should only roll if all 4 moves are Physical or Special, while Wide Guard is a Status move.